Formula/swift-protobuf.rb

```diff
@@ -0,0 +1,35 @@
+class SwiftProtobuf < Formula
+  desc "Plugin and runtime library for using protobuf with Swift"
+  homepage "https://github.com/apple/swift-protobuf"
+  url "https://github.com/apple/swift-protobuf/archive/0.9.904.tar.gz"
+  sha256 "d65970c3f17cb4a09e61102d7dce37733ad34fe723cf65795e856ec2cd126711"
+  head "https://github.com/apple/swift-protobuf.git"
+
+  depends_on :xcode => ["8.3", :build]
+  depends_on "protobuf"
+
+  conflicts_with "protobuf-swift",
```

Comment:
I think this would _(just about)_ be under 80 characters wide if on a single line? If that's the case it's probably fine not to split onto two lines like this.

Reply:
It's exactly 80 chars wide, but I think this reads cleaner.

Let me know if projects preferred syntax is single line and I can join them.

Reply:
Will defer to ILZ. My personal preference is single line where it fits inside the 80, but ILZ runs the show around here so it's his choice.

Reply:
Yes 1 line if <= 80.

Reply:
Fixed!

Also splitted "swift build" line into two, as it ran a bit longer than 80 chars. Followed the wrapping style from the  `sourcery` formula for that one.

Reply:
(no longer fits on one line because I added the word "binaries" to be consistent with other conflicts_with statements)

Reply:
Heh, was just about to check my curiosity on this 😄. Thanks for adding the notation here.